lib/matplotlib/backends/_gtk3_compat.py

```diff
@@ -40,7 +40,15 @@
 
 if gi.__name__ == "pgi" and gi.version_info < (0, 0, 11, 2):
     raise ImportError("The GTK3 backends are incompatible with pgi<0.0.11.2")
-gi.require_version("Gtk", "3.0")
+try:
+    # :raises ValueError: If module/version is already loaded, already
```

Comment:
Not shure what the leading colon is for, but heh.

Reply:
I just copy-pasted their docstring (which uses the google style docstrings)